core/chains/evm/types/models_codecgen.go

```diff
@@ -0,0 +1,3027 @@
+//go:build go1.6
+// +build go1.6
+
+// Code generated by codecgen - DO NOT EDIT.
```

Comment:
This file looks pretty gnarly. Is it possible/would it make sense to have it in an `internal/` subdirectory?

Reply:
maybe?

if so, it will be a bigger change. the generated methods need to be in the same package as the original types.

it almost works by moving blockInternal and transactionInternal to /internal/foo BUT blockInternal depends on Transaction, not transactionInternal.

 so i think we would need another translation between []transactionInternal and []Transaction

Reply:
nbd in that case

Reply:
actually, i already did it.

as a nice side effect, the code is even faster now. my guess is the my original code gen had a bunch of extra unneeded stuff.

i like the code layout more now, too